src/App.jsx

```diff
@@ -0,0 +1,45 @@
+import { useEffect, useState } from "react";
+import RestaurantButton from "./Components/RestaurantButton/index.jsx";
+import HeroText from "./Components/HeroText/index.jsx";
+
+function App() {
+  const [restaurantNames, setRestaurantNames] = useState([]);
+
+  useEffect(() => {
+    fetch("https://food-delivery-api.dev.io-academy.uk/restaurants")
+      .then((response) => response.json())
+      .then((data) => {
+        setRestaurantNames(data);
+      });
+  }, []);
+
+  return (
+    <>
+      <header className="p-4 text-center shadow-lg md:text-left">
+        <p>
+          <span className="text-cyan-500">Food</span>Delivery
+        </p>
+      </header>
+      <section className="w-full mt-4 md:px-4 h-60">
+        <div className="bg-[url(/burgers.jpg)] w-full h-full bg-cover bg-center content-center">
+          <HeroText />
+        </div>
+      </section>
```

Comment:
To me this feels a bit like a `Hero` component has been cut in half.

Half of the `Hero` is here in `App` - the part to do with the background image and the space it takes up on the page.

The other half is just the title and subtitle, in the `HeroText` component.

Really for this content to work and make sense we would need all of it together.

I dispute that this is a component at all: 

- there is no javascript logic involved in displaying a static title text and subtitle.
- we don't re-use the content or styling anywhere else.

The `HeroText` "component" is basically just a `div` - doesn't meet my threshold for being a component.

If you think the `Hero` content should be a component, then it should work together as one component.